CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Phootwork contributing guide
+
+If you're reading this document, it is likely that you have decided to contribute to this project.
+Therefore, first of all, we want to warmly thank you! The Open Source Software grows and improves
+thanks to people like you!
+
+Contributing to a project is a process with its own rules and we try to explain them along this document.
+
+## Quick start for experienced programmers
```

Comment:
`Quick Start for Experienced Programmers`

Reply:
Done